types/highcharts/index.d.ts

```diff
@@ -5420,7 +5484,16 @@ declare namespace Highcharts {
         width?: number | string;
     }
 
-    interface ScatterChart extends SeriesChart { }
+    interface ScatterChart extends SeriesChart {
+        /**
+         * For area-like series, allow the font size to vary so that small areas get a smaller font size. The default applies this effect to area-like series but not line-like series.
+         */
+        maxFontSize?: number;
```

Comment:
Are all of these just part of `SeriesChart`s? If not, why not extract them to their own interface and avoid the duplication?